pkg/bootstrap/docker/up.go

```diff
@@ -720,9 +742,14 @@ func (c *ClientStartConfig) ServerInfo(out io.Writer) error {
 		metricsInfo = fmt.Sprintf("The metrics service is available at:\n"+
 			"    https://%s\n\n", openshift.MetricsHost(c.RoutingSuffix, c.ServerIP))
 	}
+	loggingInfo := ""
+	if c.ShouldInstallLogging && c.ShouldInitializeData() {
+		loggingInfo = fmt.Sprintf("The kibana logging UI is available at:\n"+
+			"    https://%s\n\n", openshift.LoggingHost(c.RoutingSuffix, c.ServerIP))
+	}
```

Comment:
(As much a note to self as anything): this info should probably go into oc cluster status as well - see whether this PR beats #11171 or not!